tests/fixtures/hqplayer/hqpd-6.0.4-opal/restore_response.html

```
@@ -0,0 +1,22 @@
+<!--
+  provenance:
+    source: UHC-SALVAGE-UI-DATA-INTEGRATION.md and UHC-SALVAGE-BETA-DEV.md (2026-07-29 comparative salvage reports), which cite https://github.com/ohshitgorillas/hqptuner/blob/67557939ae04b157b47cb67bd651b72c3140bcdd/docs/protocol.md (section 3.6, POST /restore and the POST /config submission contract)
+    source_chain: read-via-report. The upstream file was NOT read directly for this fixture; the salvage report is the immediate source and the upstream ref is the report's own citation. Recorded because two errors in this amendment came from treating a report's citation as if it had been read: a path that does not exist at the ref given, and a superseded claim the report's own newer companion had already corrected.
+    daemon: hqplayerd 6.0.4 (Opal), persistent HTTP lane on port 8088
+    status: derived-semantics
+    date: 2026-07-29
+    playback: unknown
+    notes: The body a persistent WRITE answers with. Verified twice over: 'The 200 response body is the HTML restore page - success is confirmed by a /backup readback, never by the POST', and for POST /config 'a partial POST is silently rejected - the daemon answers HTTP 200 with Failed! in the body and writes nothing'. So an HTTP 200 here proves only that the request was received. This fixture exists so that rule is executable: the body carries no machine-readable outcome and no profile list, therefore a client must not infer success from it. The restore transport itself (multipart upload, daemon self-restart, readback polling) is issue #330's, not #322's. The page markup is representative rather than captured: what is verified is the rule that this response family carries no outcome.
+-->
+<html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)